docs/python_docs/python/Makefile

```diff
@@ -47,7 +46,7 @@ build/%: %
 html: $(OBJ)
 	mkdir -p build
 	cp Makefile_sphinx build/Makefile
-	sphinx-autogen build/api/*.rst build/api/*/*.rst   -t build/_templates/
+	sphinx-autogen build/api/*.rst build/api/**/*.rst   -t build/_templates/
```

Comment:
Is that `**` some recursive notation for file paths?

Reply:
That's the idea but actually I think the effect is the same with a single `*` .